packages/nimble-components/src/wafer-map/modules/worker-renderer.ts

```diff
@@ -0,0 +1,28 @@
+import { wrap, Remote } from 'comlink';
```

Comment:
Lets have this file name align with the api it is creating.

`create-matrix-renderer.ts`

Reply:
good point
![image](https://github.com/ni/nimble/assets/110180309/20c359df-32b2-4cdb-9387-739e5fb6b397)